test/multiarch_utils.go

```diff
@@ -106,14 +118,44 @@ func imageNamesMapping() map[string]string {
 			"stedolan/jq":                           "ibmcom/jq-s390x:latest",
 			"gcr.io/kaniko-project/executor:v1.3.0": getTestImage(kanikoImage),
 		}
+	case "ppc64le":
+		return map[string]string{
+			"gcr.io/tekton-releases/dogfooding/alpine-git-nonroot:mario": "gcr.io/tekton-releases/dogfooding/alpine-git-nonroot:latest",
+			"registry":                              getTestImage(registryImage),
+			"node":                                  "node:alpine3.11",
+			"lachlanevenson/k8s-kubectl":            getTestImage(kubectlImage),
+			"gcr.io/cloud-builders/git":             "alpine/git:latest",
+			"docker:dind":                           "docker:18.06.2-dind",
+			"docker":                                "docker:18.06.3",
+			"mikefarah/yq":                          "danielxlee/yq:2.4.0",
+			"stedolan/jq":                           "ibmcom/jq-ppc64le:latest",
+			"gcr.io/kaniko-project/executor:v1.3.0": getTestImage(kanikoImage),
+			"quay.io/rhpipeline/skopeo:alpine":      "ibmcom/skopeo-ppc64le",
```

Comment:
This image is not used in tests any more

Reply:
Updated